.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
This file is a .gitignore file. If you put one of these inside a Git Repository, then Git will ignore the files you specify. We need this because we have added NPM and we don't want our dependencies to be checked into our repository (users should perform npm install to get them). See my comments on the package.json file for more info.